Modulo1-Fundamentos/6-HTML_e_CSS:_Forms,_Flexbox_e_Responsivo/6.1-HTML-&-CSS-Forms/preventDefault/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <a href="www.betrybe.com" target="_blank" id="href">TRYBE</a>
```

Comment:
```suggestion
    <a href="https://www.betrybe.com" target="_blank" id="href">TRYBE</a>
```
Opaa belê? Links externos precisam começar com http ou https://